api/services/packages.ts

```diff
@@ -51,3 +51,12 @@ export const useViewPackage = (packageId: string) => {
         queryKey: [baseQueryKey],
     });
 }
+
+export const useEditPackage = ( editedPackage: PaginatedResponse<Package>, id?: string) => {
+
+return useAppMutation<PaginatedResponse<Package>>({
+    apiRoute: routes.packages.editActivePackages(id),
+    method: 'PUT',
+    body: editedPackage
```

Comment:
you should json stringify before you make a post or patch request. this would throw an error